Formula/aliyunpan.rb

```diff
@@ -0,0 +1,29 @@
+class Aliyunpan < Formula
+  desc "Command-line client tool for Alibaba aDrive disk"
+  homepage "https://github.com/tickstep/aliyunpan"
+  url "https://github.com/tickstep/aliyunpan/archive/refs/tags/v0.2.7-1.tar.gz"
```

Comment:
Is the version `0.2.7` or `0.2.7-1`?

Reply:
the version is 0.2.7
the tag 0.2.7 has been used before, so we make an new tag 0.2.7-1 for submit this brew PR

Reply:
What is the difference between the `v0.2.7-1` and `v0.2.7` tags?

Reply:
In order to meet the testing requirements of brew, we follow brew guide to add a function for testing in v0.2.7-1. Since the main functions of this tool are not affected, we don't want to release a new version and impact our users.